MobileTraining/MobileTraining/components/Product/ProductView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ProductView.swift
+//  MobileTraining
+//
+//  Created by Svetoslav Donkov on 14.05.23.
+//
+
+import SwiftUI
+
+struct Product: Identifiable {
+    let id: Int
+    let imageName: String
+    let title: String
+    let description: String
+    let details: String
+    let price: Double
+}
+
+let sampleProduct = Product(
```

Comment:
I know this is a mock but you can make yourself a view model and place it there, as to prepare for the backend integration.